README.md

```diff
@@ -16,7 +16,9 @@ the square brackets (for multiple-choice questions).
 What does it mean for an HTML tag to be "semantic"? Give an example of a short snippet of HTML written using semantic tags and non-semantic tags.
 
 ```text
-Your answer here.
+Semantic tags have direct purposes related to their elements whereas non-semantic are more general and can be used for multiple purposes.
+Semantic tag example: Images - <img src="myimage.jpg" alt="alt text">
+Non-semantic tag example: <div class="snip">some text here</div>
```

Comment:
I'd say more specifically semantic tags tell you what the text in the element *does*.